include/freerdp/update.h

```diff
@@ -243,6 +243,11 @@ struct rdp_update
 	rdpPcap* pcap_rfx;
 	BOOL initialState;
 
+	/* if autoCalculateBitmapData is set to TRUE, the server automatically 
+	 * fills BITMAP_DATA struct members: flags, cbCompMainBodySize and cbCompFirstRowSize.
+	*/
+	BOOL autoCalculateBitmapData;
+
```

Comment:
would be nice to move that new field at the end of the struct for binary compatibility